drivers/source/AnalogIn.cpp

```diff
@@ -54,6 +54,21 @@ unsigned short AnalogIn::read_u16()
     return ret;
 }
 
+float AnalogIn::read_voltage()
+{
+    return (this->read() * this->_vref);
```

Comment:
Excess `()`. We wouldn't normally use the explicit `this->` either, and the rest of the file doesn't.